src/app/core/component/container/container.component.html

```diff
@@ -0,0 +1,5 @@
+<p>
+  container works!
+</p>
+<app-commit-table></app-commit-table>
```

Comment:
Met des "s" a componentS et containerS

Reply:
J'aurai mis containers au meme niveau que components dans l'idee du concept de smart and dumb components mais comme tu veux, par contre faudras l'expliquer dans le readme